py-polars/tests/unit/operations/namespaces/temporal/test_datetime.py

```diff
@@ -1100,10 +1101,11 @@ def test_mean(
     values: list[TemporalLiteral | None], expected_mean: TemporalLiteral | None
 ) -> None:
     s = pl.Series(values)
-    assert s.dt.mean() == expected_mean
 
-    if s.dtype in (pl.Datetime, pl.Duration, pl.Time):
-        assert s.mean() == expected_mean
+    assert s.mean() == expected_mean
+
+    with pytest.deprecated_call():
```

Comment:
Could you please rewrite those tests to use `Series.mean` instead of `Series.dt.mean`? And leave only a single test that checks the deprecation and the existing functionality for `dt.mean`. Same for `median`.

Reply:
I decreased the coverage slightly of the `dt.mean` deprecation tests, since the code simply calls the underlying `Series.mean` now anyway and the test is mainly to ensure that the deprecation works. Putting everything into one test would have been a bit unwieldy since it was multiple functions/tests. Let me know if this is reasonable or if you want me to put the full coverage back in.